src/main/resources/com/cloudbees/hudson/plugins/folder/computed/FolderComputation/index.jelly

```diff
@@ -33,7 +34,7 @@ THE SOFTWARE.
                     (<i:formatDate value="${it.timestamp.time}" type="both" dateStyle="medium" timeStyle="medium"/>)
                 </j:if>
             </t:buildCaption>
-            <table style="margin-top: 1em; margin-left:1em;">
+            <fo:blockWrapper>
```

Comment:
do you have any "before/after" screenshot? With this change we're losing the "style" attribute with the margin values

Reply:
I clicked through everywhere and couldn't see anything that looked off.

I don't have time right now to do that, feel free to add.

This is a fix for preventing creation of a branch source completely...

Reply:
I will test myself this PR and will be back then